apps/services/schemas.py

```diff
@@ -105,7 +105,7 @@ class HostelAccomodationViewOut(BaseOutSchema):
     university_name: str
     organisation_code: str
     payment_recognation: str  # TODO spelling error
-    commandant_full_name: str
+    commandant_full_name: Dict[str, str]
```

Comment:
Should be an embedded schema, that describes this structure

Reply:
example?

Reply:
fixed